core/tests/webdriverio_desktop/extensions.js

```diff
@@ -40,7 +40,7 @@ describe('rich-text components', function () {
     explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
   });
 
-  it('should display correctly', async function () {
+  it('should display correctly part 1', async function () {
```

Comment:
Please use a clearer description, not "part 1".

Reply:
This was just for testing purposes, reverted to the original test name